src/plugins/data/common/search/expressions/kibana_context.ts

```diff
@@ -61,7 +63,7 @@ export const kibanaContextFunction: ExpressionFunctionKibanaContext = {
       }),
     },
     timeRange: {
-      types: ['string', 'null'],
+      types: ['timerange', 'null'],
```

Comment:
I guess these two type changes are breaking changes, so we need a breaking change note.

Reply:
as we never store the expression (except in canvas), nor expose expression to the user (except in canvas) and this function is not used in canvas I think we are good